android/quest/src/test/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireViewModelTest.kt

```diff
@@ -1144,4 +1146,73 @@ class QuestionnaireViewModelTest : RobolectricTest() {
       bundle.entry.any { it.resource is Basic && it.resource.id == "basic-resource-id" },
     )
   }
+
+  @Test
+  fun isRelatedEntityLocationMetaTagAddedToExtractedResource() = runBlocking {
```

Comment:
Rename this test case to align with the others. Maybe something like
```suggestion
  fun testSaveExtractedResourcesAddsRelatedEntityLocationMetaTagToExtractedResource() = runBlocking {
```